### PR TITLE
fix: Exercise 06 deprecated --serviceaccount flag fix

### PR DESCRIPTION
**Pull request by addisonjones-gc**
Replaced the deprecated --serviceaccount flag with another example of a one liner.

### CODE REVIEW

**Comment by bmuschko** [comment]
What version of Kubernetes deprecates the flag? I tried with v1.22.3 and I don't get a deprecation warning.

**Comment by addisonjones-gc** [comment]
Looks like I'm on 1.23.4

Here's the message I'm getting: 

```
Flag --serviceaccount has been deprecated, has no effect and will be removed in 1.24.
```

File: exercises/06-using-service-account/solution/solution.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ secrets:
 Next, you can create a new Pod and assign the service account to it.
 
 ```shell
-$ kubectl run backend --image=nginx --restart=Never --serviceaccount=backend-team
+$ kubectl run backend --image=nginx --restart=Never --overrides='{ "spec": { "serviceAccount": "backend-team" } }'
```
Comment (bmuschko) [inline exercises/06-using-service-account/solution/solution.md]: The attribute `serviceAccount` in the PodSpec is [deprecated](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#pod-v1-core).

> ServiceAccount is a depreciated alias for ServiceAccountName. Deprecated: Use serviceAccountName instead.

Can you please change this to `serviceAccountName`?

**Review by bmuschko — CHANGES_REQUESTED** [review]

File: exercises/06-using-service-account/solution/solution.md (hunk 1 above)
Comment (addisonjones-gc) [reply exercises/06-using-service-account/solution/solution.md]: Corrected to `serviceAccountName`. Thanks!

**Comment by bmuschko** [comment]
Perfect, thanks!

**Pull request merged**